RecoTauTag/HLTProducers/interface/L1TJetsMatching.h

```diff
@@ -37,7 +37,7 @@
 
 #include "Math/GenVector/VectorUtil.h"
 #include "DataFormats/HLTReco/interface/TriggerTypeDefs.h"
-#include "FWCore/Utilities/interface/EDMException.h"
+#include "FWCore/Utilities/interface/Exception.h"
 #include "DataFormats/JetReco/interface/PFJet.h"
 
```

Comment:
In the header includes, remove `<map>` (unused?), and add `<utility>` (for std::pair), `<tuple>` (for std::tuple), and `<string>`.

Reply:
done